lib/Backend/Lower.cpp

```diff
@@ -8181,7 +8184,7 @@ Lowerer::LowerStElemI(IR::Instr * instr, Js::PropertyOperationFlags flags, bool
 ///
 ///----------------------------------------------------------------------------
 IR::Instr *
-Lowerer::LowerLdElemI(IR::Instr * instr, IR::JnHelperMethod helperMethod, bool isHelper)
+Lowerer::LowerLdElemI(IR::Instr * instr, IR::JnHelperMethod helperMethod, bool isHelper, IR::LabelInstr * labelBailOut)
```

Comment:
Done.

---
In reply to: [67767447](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67767447) [](ancestors = 67767447)